components/create-proposal.tsx

```diff
@@ -50,10 +56,81 @@ export default function NewProposal({
 
 
   function handleChangeDistrib(params: { [key: string]: number }): void {
-    setDistrib((prevState) => ({
+    setDistrib((prevState) => {
+      handleCheckDistrib({
+        ...prevState,
+        ...params,
+        })
+      return({
       ...prevState,
       ...params,
-    }));
+      })
+  });
+  }
+
+  async function loadProposalsMeta() {
+    if (!issueId)
+      return;
+
+    const scIssueId = await BeproService.network.getIssueByCID({issueCID: issueId}).then(({_id}) => _id);
+    const pool = [];
+
+    for (const meta of mergeProposals as ProposalData[]) {
+      const { scMergeId, pullRequestId } = meta;
+      if (scMergeId) {
+        const merge = await BeproService.network.getMergeById({merge_id: scMergeId, issue_id: scIssueId});
+        pool.push({...merge, pullRequestId } as Proposal)
+      }
+    }
+
+    setProposals(pool);
+  }
+
+  function handleCheckDistrib(obj: object) {
+    var currentAmount = sumObj(obj)
+    if (currentAmount === 100){
+     const { id }  = pullRequests.find(
+        (data) => data.githubId === currentGithubId
+      )
+      var currentDistrbuition = {
+        currentPrId: id,
+        prAmounts: participants.map(
+          (items) =>  ((amountTotal * obj[items.githubHandle])/100).toString()
+        )
+      }
+      const equals = (a, b) => JSON.stringify(a) === JSON.stringify(b);
+      if(
+        proposals.find(
+          (data) =>
+            equals(currentDistrbuition.prAmounts, data.prAmounts) === true &&
+            data.pullRequestId === currentDistrbuition.currentPrId
+        )
+      ){
+        setWarning(' ')
+        setError('')
+        setSuccess('')
+      }else {
+        setError('')
+        setSuccess(' ')
+        setWarning('')
+      } 
+   }
+   if (currentAmount > 0 && currentAmount < 100){
```

Comment:
these three conditions can be simplified,

```
setError(currentAmount > 0 ? ` ` : ``)
setSuccess(``)
setWarning(``)
```

Also, double-check you're not missing any error messages.